### DISCUSSION TITLE
no enemy vehicles in city garrison

### DISCUSSION THREAD

**Discussion by SOF-Ice** [Vdauphin/HeartsAndMinds#1438] [Question and answer] [answered]
Hi !

I am using 3CB Factions for OPFOR faction.

When i want to test, the mission run good. But there are no enemies vehicles in city garrisons. Only infantry

Im using the latest version of H&M and 3CB Factions

I can see OPFOR vehicles only when they spawn as random patrol and in some side missions

whereas, i checked "enemy used tanks", "enemy armed vehicles in hideout", ...

i guess i do something rong but i dont know what

**Comment by Vdauphin** [comment, marked as answer]
Hello,
This has been like that since the start see: https://github.com/Vdauphin/HeartsAndMinds/discussions/1427#discussioncomment-3888901
Cheers

**Reply by SOF-Ice** [reply]
Thanks for reply.
I ever see this post.

But i dont know what im supposed to change in this file to solve the issue

**Reply by Vdauphin** [reply]
> But i dont know what im supposed to change in this file to solve the issue

There are no issue as it is like that by design. When player enter a hideout, reinforcement will be send to it and then remain in it (if you had activate  "enemy armed vehicles in hideout"). But vehicles will never spawn in first activation inside the city. There are send from an other city

You can check here: https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/city/activate.sqf#L247-L249

The `btc_city_fnc_send` will send vehicle from nearby city to hideout

**Reply by SOF-Ice** [reply]
Owww ok ! I understand ...

My bad 

Thanks for all
